modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/SslHandlerFactory.java

```diff
@@ -89,7 +89,7 @@ private KeyStore getKeyStore(String type, String storePath, String password)
     private Path getPath(String storePath) {
         Path path = Paths.get(storePath);
 
-        if (Files.notExists(path)) {
+        if (!path.toFile().exists()) {
```

Comment:
What is the advantage of this over the other? Just need to understand.

Reply:
There is a performance issue with that way of checking for existence. Refer https://rules.sonarsource.com/java/tag/java8/RSPEC-3725